.github/workflows/build.yml

```diff
@@ -44,16 +49,21 @@ jobs:
           git config user.password "${ROBOT_TOKEN}"
           echo "GIT_USER=percona-platform-robot:${ROBOT_TOKEN}" >> $GITHUB_ENV
 
+  build:
```

Comment:
Not sure we need this as a separate step. We could continue running steps in the same job. Whenever a step fails, the whole job fails. Unless you want to continue despite a failing job, this is probably not necessary.

Reply:
The whole point of this PR was to separate into jobs. There was no other change. The reason is that you can't see what stage has failed without drilling down. With jobs, you see immediately which part of setup/build/(spell/grammar) check has a problem.

Reply:
It's an overkill, since every job starts a fresh operating system, which slows down the workflow and burns the resources. 

You can see what stage has failed without drilling down. Example attached.

<img width="1401" alt="Screenshot 2021-02-12 at 12 06 24" src="https://user-images.githubusercontent.com/81549/107748870-c53b8b80-6d2a-11eb-8227-4b12abb1ed36.png">

But I leave it up to you :)